src/pages/users/[id]/index.tsx

```diff
@@ -19,56 +26,62 @@ const UserProfilePage = () => {
   }
 
   return (
-    <div className="p-4">
-      <div className="bg-white dark:bg-gray-800 shadow overflow-hidden sm:rounded-lg">
-        <div className="px-4 py-5 sm:px-6">
-          <h3 className="text-lg leading-6 font-medium text-gray-900 dark:text-white">
-            User Profile
-          </h3>
-          <p className="mt-1 max-w-2xl text-sm text-gray-500 dark:text-gray-400">
-            Personal details and application.
-          </p>
-        </div>
-        <div className="border-t border-gray-200 dark:border-gray-700">
-          <dl>
-            <div className="bg-gray-50 dark:bg-gray-900 px-4 py-5 grid grid-cols-1 gap-4 sm:grid-cols-3">
-              <dt className="text-sm font-medium text-gray-500 dark:text-gray-400">
-                Full name
-              </dt>
-              <dd className="mt-1 text-sm text-gray-900 dark:text-white sm:col-span-2">
-                {user.name}
-              </dd>
-            </div>
-            <div className="bg-white dark:bg-gray-800 px-4 py-5 grid grid-cols-1 gap-4 sm:grid-cols-3">
-              <dt className="text-sm font-medium text-gray-500 dark:text-gray-400">
-                Encryption Public Key
-              </dt>
-              <dd className="mt-1 text-sm text-gray-900 dark:text-white sm:col-span-2">
-                {user.encPk}
-              </dd>
+    <div>
+      <AppBackHeader />
+      <div className="flex flex-col gap-6">
+        <div className="flex gap-6 items-center">
+          <div className="h-32 w-32 rounded bg-slate-200"></div>
+          <div className="flex flex-col gap-1">
+            <h2 className=" text-xl font-gray-12 font-light">{user.name}</h2>
+            <div className="flex items-center gap-1">
+              <Icons.checkedCircle />
+              <span className="text-sm font-light text-white">
+                Connected XXX
+              </span>
             </div>
+            <Button onClick={() => router.push(`/users/${id}/share`)}>
```

Comment:
This should only appear if you haven't already connected right? Not by default?

Reply:
I think this is a design discussion - i'm thinking its totally valid for people to reshare if they want to say update the information they send to someone, but we could also disable this